ynr/apps/bulk_adding/management/commands/bulk_adding_import_from_council_csv.py

```diff
@@ -0,0 +1,19 @@
+from django.core.management.base import BaseCommand
+
+from bulk_adding.helpers import CSVImporter
+from elections.models import Election
+
+
+class Command(BaseCommand):
+    help = "Create a RawBallotInput model from a council provided CSV"
+
+    def add_arguments(self, parser):
+        parser.add_argument("--file", action="store", required=True)
+        parser.add_argument("--election", action="store", required=True)
+
+    def handle(self, *args, **options):
+        ballot = Election.objects.get(slug=options["election"])
```

Comment:
This isn't a ballot, its an Election (remember in YNR those aren't the same thing)